platforms/js/opencv_js.config.py

```diff
@@ -55,8 +55,26 @@
               'BFMatcher': ['isMaskSupported', 'create'],
               '': ['drawKeypoints', 'drawMatches', 'drawMatchesKnn']}
 
-calib3d = {'': ['findHomography', 'calibrateCameraExtended', 'drawFrameAxes', 'estimateAffine2D', \
-                'getDefaultNewCameraMatrix', 'initUndistortRectifyMap', 'Rodrigues', \
-                'solvePnP', 'solvePnPRansac', 'solvePnPRefineLM']}
+calib3d = {
+    '': [
+        'findHomography',
+        'calibrateCameraExtended',
+        'drawFrameAxes',
+        'estimateAffine2D',
+        'getDefaultNewCameraMatrix',
+        'initUndistortRectifyMap',
+        'Rodrigues',
+        'solvePnP',
+        'solvePnPRansac',
+        'solvePnPRefineLM',
+        'projectPoints',
+
+        # cv::fisheye namespace
+        'fisheye_initUndistortRectifyMap',
+        'fisheye_projectPoints',
+    ],
+}
```

Comment:
Now this is the only list in the file that's formatted this way

Reply:
This file is a sample of opencv.js configuration.
Users of custom opencv.js builds already have own configurations, so we need guarantee compatibility for them.

Reply:
1 item per-line formatting is for easy merging between branches (it is already partially used on the master/4.x branch).